app/models/evidence/forms/income.rb

```diff
@@ -0,0 +1,27 @@
+module Evidence
+  module Forms
+    class Income < ::FormObject
+
+      include ActiveModel::Validations::Callbacks
+
+      def self.permitted_attributes
+        {
+          amount: String
+        }
+      end
+
+      define_attributes
+
+      before_validation :format_amount
```

Comment:
I'm using this callback now, but when it comes to persisting this data, I'll remove the callback and I'll just reuse the **format_amount** method to do the rounding of the amount before saving it.

Something like this:

```ruby
def save
   if valid?
     persist!
     true
   else
     false
   end
 end

private

def persist!
  @evidence = Evidence.create!(income: format_amount(amount))
end
```